src/locales/en.ts

```diff
@@ -713,18 +713,32 @@ const en = {
         },
         notebook: {
           title: 'Variant Workbench',
+          dataStudio: 'Data Studio',
+          tooltip: {
+            title: 'CAVATICA VWB — Data Studio',
+            part1: 'Analyze Kids First’s variant data in Cavatica’s',
+            part2:
+              'for enhanced data manipulation.Once your files are copied into a Cavatica project, you can explore and combine Kids First participant clinical data, variant annotations, and public external variant databases (such as Ensembl, gnomAD, dbSNFP, OMIM) in JupyterLab with PySpark to conduct statistical analyses, integrate multi-omics data, generate predictive models, and create compelling visualizations.',
```

Comment:
Il manque un espace rien de fou 😄 
```suggestion
              'for enhanced data manipulation. Once your files are copied into a Cavatica project, you can explore and combine Kids First participant clinical data, variant annotations, and public external variant databases (such as Ensembl, gnomAD, dbSNFP, OMIM) in JupyterLab with PySpark to conduct statistical analyses, integrate multi-omics data, generate predictive models, and create compelling visualizations.',
```